packages/core/src/components/Table/TableHeaderCell/TableHeaderCell.tsx

```diff
@@ -37,14 +38,24 @@ const TableHeaderCell: VibeComponent<ITableHeaderCellProps, HTMLDivElement> = fo
     },
     ref
   ) => {
+    const [isHovered, setIsHovered] = useState<boolean>(false);
+    const ariaSort = getAriaSort(sortState);
+    const isSortActive = onSortClicked && ariaSort !== "none";
+    const shouldShowSortIcon = ariaSort !== "none" || isHovered;
+
     return (
       <div
         ref={ref}
         id={id}
-        className={cx(styles.tableHeaderCell, className)}
+        className={cx(styles.tableHeaderCell, { [styles.sortActive]: isSortActive }, className)}
         data-testid={dataTestId || getTestId(ComponentDefaultTestId.TABLE_HEADER_CELL, id)}
         role="columnheader"
-        aria-sort={getAriaSort(sortState)}
+        onMouseOver={() => setIsHovered(true)}
+        onMouseLeave={() => setIsHovered(false)}
+        onFocus={() => setIsHovered(true)}
+        onBlur={() => setIsHovered(false)}
+        aria-sort={onSortClicked ? ariaSort : undefined}
+        tabIndex={onSortClicked ? 0 : undefined}
```

Comment:
null instead?

Reply:
on purpose, since I do not want the prop to be defined, it is the same as we're not passing a prop at all
for `aria-sort` for example, I do not want to show aria-sort in the DOM since a screen-reader can think it is sortable
for `tabIndex`, null is not a correct value, and I want to use the default